charts/spilo/templates/statefulset.yaml

```diff
@@ -65,6 +65,21 @@ spec:
         {{- if eq .Values.continuousArchive.storage "s3"}}
         - name: WAL_S3_BUCKET
           value: {{ .Values.walE.s3Bucket | quote }}
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom: 
+            secretKeyRef: 
+              name: {{ .Values.continuousArchive.s3.secretName }}
+              key: {{ .Values.continuousArchive.s3.awsAccessKeyIdName }}
```

Comment:
```suggestion
              key: AWS_ACCESS_KEY_ID
```
or 
```suggestion
              key: aws_access_key_id
```

Reply:
I think this way could be more configurable if the project already has the secret, but it is also true that it is bit confusing.

Reply:
I updated the key names in the secret to be same as the env variables